test/unit/db/trace.test.js

```diff
@@ -5,13 +5,13 @@
 
 'use strict'
 
-const tap = require('tap')
```

Comment:
you have some tests that are callback based in here. they have to pass the `end()` cb.  it's the 2nd arg to tests